yt_dlp/extractor/mixch.py

```diff
@@ -62,20 +69,44 @@ class MixchArchiveIE(InfoExtractor):
             'id': '421',
             'title': '96NEKO SHOW TIME',
         }
+    }, {
+        'url': 'https://mixch.tv/archive/1213',
+        'skip': 'paid video, no DRM. expires at Dec 31, 2023',
+        'info_dict': {
+            'id': '1213',
+            'title': '【特別トーク番組アーカイブス】Merm4id×燐舞曲 2nd LIVE「VERSUS」',
+            'live_status': 'not_live',
+            'release_timestamp': 1701421200,
+        }
+    }, {
+        'url': 'https://mixch.tv/archive/1214',
+        'only_matching': True,
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        webpage = self._download_webpage(url, video_id)
 
-        html5_videos = self._parse_html5_media_entries(
-            url, webpage.replace('video-js', 'video'), video_id, 'hls')
-        if not html5_videos:
+        info_json, urlh = self._download_json_handle(
+            f'https://mixch.tv/api-web/archive/{video_id}', video_id, expected_status=(401))
+        if urlh.status == 401:
             self.raise_login_required(method='cookies')
-        infodict = html5_videos[0]
-        infodict.update({
-            'id': video_id,
-            'title': self._html_search_regex(r'class="archive-title">(.+?)</', webpage, 'title')
-        })
+            return
+
+        release_timestamp = None
+        month, day, hour, min = self._search_regex(
+            r'(?P<month>\d{1,2})月(?P<day>\d{1,2})日\([月火水木金土日]\)(?P<hour>\d{1,2}):(?P<min>\d{1,2})',
+            traverse_obj(info_json, ('archive', 'start')), name='start datetime',
+            default=(None, None, None, None), group=('month', 'day', 'hour', 'min'))
+        if month and day and hour and min:
+            year = (datetime_from_str('today') + datetime.timedelta(hours=9)).strftime('%Y')
```

Comment:
Entries are always from the current year?

Reply:
The content of the JSON is:

```JSON
{
  "archive": {
    "id": 42,
    "title": "...",
    "thumbnailURL": "https://.../thumbnail.png",
    "start": "12月1日(金)18:00",
    "end": "12月31日(日)23:59",
    "archiveURL": "https://.../video.m3u8"
  },
  "isIndefinitePeriod": false
}

```

No years at all.

I guess we may have a way to figure out if the start, the end and now are in the same year...?

Reply:
If this entry was from 2023, could you check again what the API returns now in 2024?
If it was from 2022 or prior, or it is still the same, I would be tempted to say we should not include a release timestamp as we just don't know the date.

Reply:
> If this entry was from 2023, could you check again what the API returns now in 2024?

IIRC, the website quickly removed the data about expired events, so I'm not able to get anything.

> I would be tempted to say we should not include a release timestamp as we just don't know the date.

I agree. Changes are committed.